### DISCUSSION TITLE
Rsbuild - Build failure when migrating from Vue CLI

### DISCUSSION THREAD

**Discussion by exploitationExplorers** [web-infra-dev/rsbuild#3765] [Q&A]
[package.json](https://github.com/user-attachments/files/17443368/package.json)

Module build failed:
  ├─▶   ×   × Expected '>', got 'id'
  │     │    ╭─[D:\qushiyun\园区\e-park-web\src\App.vue:1:1]
  │     │  1 │ <div id="app">
  │     │    ·      ──
  │     │  2 │   <router-view />
  │     │  3 │ </div>;
  │     │    ╰────
  │     │
  │   
  ╰─▶ Syntax Error/nFile: D:\qushiyun\园区\e-park-web\src\components\Breadcrumb\index.vue?vue&type=template&id=b50ef614&scoped=true:1:1
  × Module build failed:

**Comment by 9aoy** [comment]
Please provide a reproduction repo.
